app/scripts/controllers/eventRegistrations.js

```diff
@@ -317,38 +317,17 @@ angular.module('confRegistrationWebApp')
 
       var showGroupDialogOptions = {
         templateUrl: showGroupModalTemplate,
+        controller: 'showGroupModalCtrl',
```

Comment:
I still think it would be great if these didn't have to share $scope and could use an Angular 1.5 component which ties the controller and template together in one place.